generators/utilities/src/main/scala/Simulator.scala

```diff
@@ -82,26 +85,31 @@ object GenerateSimFiles extends App with HasGenerateSimConfig {
     out.close()
   }
   def resources(sim: Simulator): Seq[String] = Seq(
-    "/testchipip/csrc/SimSerial.cc",
-    "/testchipip/csrc/SimDRAM.cc",
-    "/testchipip/csrc/mm.h",
-    "/testchipip/csrc/mm.cc",
-    "/testchipip/csrc/mm_dramsim2.h",
-    "/testchipip/csrc/mm_dramsim2.cc",
-    "/csrc/SimDTM.cc",
-    "/csrc/SimJTAG.cc",
-    "/csrc/remote_bitbang.h",
-    "/csrc/remote_bitbang.cc",
     "/vsrc/EICG_wrapper.v",
-  ) ++ (sim match { // simulator specific files to include
-    case VerilatorSimulator => Seq(
-      "/csrc/emulator.cc",
-      "/csrc/verilator.h",
-    )
-    case VCSSimulator => Seq(
-      "/vsrc/TestDriver.v",
-    )
-  })
+    ) ++ (sim match {
```

Comment:
Do we need this empty `Seq` at the beginning?

Reply:
Sorry? Are you referring to L90? If so, yes, you want to not include the simulation files (or you have to filter them out in the `Makefile` when using `sim_common_files`)

Reply:
Oh this has changed I think since I looked at it before, or I misread before.
I thought it was:
```
Seq(
) ++ (sim match {
...
```